### PR TITLE
[routing] Fixed connectivity checking code.

### PR DESCRIPTION
**Pull request by ygorshenin**
https://jira.mail.ru/browse/MAPSME-2423

@mpimenov PTAL

### CODE REVIEW

**Comment by ygorshenin** [comment]
@bykoianko PTAL

**Comment by mpimenov** [comment]
LGTM

File: routing/road_graph_router.cpp [hunk 1]
```diff
@@ -69,27 +69,28 @@ bool CheckGraphConnectivity(IRoadGraph const & graph, Junction const & junction,
...
-      return true;
-
     edges.clear();
-    graph.GetOutgoingEdges(node, edges);
-    for (Edge const & e : edges)
-      q.push(e.GetEndJunction());
+    graph.GetOutgoingEdges(u, edges);
+    for (Edge const & edge : edges)
+    {
+      Junction const v = edge.GetEndJunction();
```
Comment (bykoianko) [inline routing/road_graph_router.cpp]: Junction const & v?

**Comment by bykoianko** [comment]
LGTM after minor fix

File: routing/road_graph_router.cpp (hunk 1 above)
Comment (ygorshenin) [inline routing/road_graph_router.cpp]: Done

**Comment by ygorshenin** [comment]
Merge please.

**Pull request merged**